.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install --force
+          && go get -u ./...
```

Comment:
`go test` if fully compatible with vendoring. It's kinda funny the way it does it. When the code is compiled to be tested, the toolchain will pull the dependencies from _vendor/_, as expected. Then, when `go test` runs the tests, it will actually recurse into the folders in _vendor/_, but because `govendor` removes all of the test files from dependencies, no tests are run.

In essence, it matches the behaviour of un-vendored code.